frontend/src/tournaments/LeaderboardTab.tsx

```diff
@@ -114,30 +136,85 @@ const LeaderboardTab = () => {
                         select
                         label='Time Control'
                         value={timeControl}
+                        color='primary'
                         onChange={(e) => setTimeControl(e.target.value as TimeControl)}
                     >
-                        <MenuItem value={'blitz'}>Blitz</MenuItem>
-                        <MenuItem value={'rapid'}>Rapid</MenuItem>
-                        <MenuItem value={'classical'}>Classical</MenuItem>
+                        <MenuItem value={'blitz'}>
```

Comment:
Can we change the value to `TimeControlType.Blitz` and do the same for the other options?

Reply:
sure